hack/boilerplate/boilerplate.Dockerfile.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright YEAR The Machine Controller Authors.
+# Copyright 2019 The Machine Controller Authors.
```

Comment:
YEAR needs to be kept as `YEAR` because this will be replaced by the year in the file. The year denotes the year the file was created in.

Reply:
I guess `The Machine Controller` authors needs to be updated to `The Loodse Machine Controller` authors.

Reply:
The script won't automatically update the license headers, it only verifies it :see_no_evil: 

You can use the code in https://github.com/kubernetes/repo-infra/pull/82 to automatically generate it for the time being :grimacing:

Reply:
Hehe, that's happening when only wanting to change XXX in the CONTRIBUTING.md. IMHO the "Loodse" part should be an extra change.